src/Microsoft.DotNet.Build.Tasks/Targets/toolruntime.targets

```diff
@@ -24,9 +24,22 @@
     <ToolHostCmd Condition="'$(COMPLUS_InstallRoot)' != ''">(set COMPLUS_InstallRoot=) &amp; $(ToolHostCmd)</ToolHostCmd>
   </PropertyGroup>
 
+  <!-- If BUILDTOOLS_TOOLS_RUNTIME is set, then we will use that as the tool and test runtime, else we will restore it from NuGet. -->
+  <Target Name="UseBuildToolsRuntimeFromEnvVariable"
```

Comment:
Currently this change will not enable that either, but we should be careful about mixing and matching these components. I agree we need to have a way to override but I'm not sure we want people to do it in a piece-wise fashion.